src/_rules/list.js

```diff
@@ -0,0 +1,28 @@
+import { entriesToCss } from '@unocss/core';
+
+const listItemStyles = entriesToCss(Object.entries({
+  position: 'relative',
+  'padding-left': 'var(--w-space-3)',
```

Comment:
I don't think the `space` values are ready for use?

Reply:
Perhaps not... 😄  How should we handle it for now?

Reply:
I would just set it to static pixels for now like the heights and widths.

Reply:
Should we use "TODOs" in these places? Or would that make it too todo cluttered at this time? 😅

Reply:
Updated with todos.